include/sys/cbprintf_cxx.h

```diff
@@ -57,6 +58,7 @@ static inline int z_cbprintf_cxx_is_pchar(const volatile wchar_t *, bool const_a
 template < typename T >
 static inline int z_cbprintf_cxx_is_pchar(T arg, bool const_as_fixed)
 {
+	ARG_UNUSED(arg);
 	_Pragma("GCC diagnostic push")
```

Comment:
can you put ARG_UNUSEDs next to each other? You can move `ARG_UNUSED(const_as_fixed);` up as well.